src/ngraph/runtime/interpreter/int_backend.hpp

```diff
@@ -312,6 +314,11 @@ class ngraph::runtime::interpreter::INTBackend : public Backend
                                    avg_pool->get_include_padding_in_avg_computation());
             break;
         }
+        case OP_TYPEID::GenerateMask:
+        {
+            throw ngraph_error(
+                "GenerateMask is an experimental op that's only supported on CPU backend");
```

Comment:
I would've preferred to put it in the `experimental` directory if this is the case, but I shan't fight it.

Reply:
Thanks for doing this.